Documentation/project-docs/dogfooding.md

```diff
@@ -14,26 +14,26 @@ this experience. Make sure to consult this document often.
 
 3. Reminder: if you are using a local copy of the dotnet CLI, take care that when you type `dotnet` you do not inadvertently pick up a different copy that you may have in your path. On Windows, for example, if you use a Developer Command Prompt, a global copy may be in the path, so use the fully qualified path to your local `dotnet`. If you receive an error "The current .NET SDK does not support targeting .NET Core 2.1." then you may be executing an older `dotnet`.
 
-After setting up dotnet you can verify you are using the newer version by executing `dotnet --info` -- the version should be greater than 2.2.0-*  (dotnet CLI is currently numbered 2.2.0-* not 2.1.0-* ). Here is an example output at the time of writing:
+After setting up dotnet you can verify you are using the newer version by executing `dotnet --info` -- the version should be greater than 2.1.300-*  (dotnet CLI for .NET Core 2.1 is currently numbered 2.1.300-*). Here is an example output at the time of writing:
```

Comment:
Make the `*`s escaped or add backticks around the version numbers, so that it is not randomly flipping to italics?

Reply:
Done. thx.